vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/ComboBoxBase.java

```diff
@@ -1130,4 +1230,9 @@ protected void runBeforeClientResponse(SerializableConsumer<UI> command) {
         getElement().getNode().runWhenAttached(ui -> ui
                 .beforeClientResponse(this, context -> command.accept(ui)));
     }
+
+    private void initConnector() {
+        getElement().executeJs(
+                "window.Vaadin.Flow.comboBoxConnector.initLazy(this)");
```

Comment:
As we initialize `comboBoxConnector` in this file, the `@JsModule("./comboBoxConnector.js")` should be probably moved to `ComboBoxBase` as well, otherwise we would have to have it imported twice.